app/move/validators.js

```diff
@@ -0,0 +1,14 @@
+const { Controller } = require('hmpo-form-wizard')
+
+module.exports = {
+  time(value) {
+    return (
+      value === '' ||
+      Controller.validators.regex(
+        value,
+        /^([0-9]|0[0-9]|1[0-9]|2[0-3]):([0-5][0-9]) ?([AaPp][Mm])?$/
+      )
+      // Controller.validators.regex(value, /^\d{1,2}:\d{2}([ap]m)?$/)
```

Comment:
Maybe remove the commented code; also, you have to read the regex to figure out which format of time this will allow. Maybe add a comment with an example of the desired format?

Reply:
Have we recorded why we're allowing just this format rather than more flexible parsing such as provided by https://sugarjs.com/dates/ ?

Reply:
So we have a "formatter" and a "validator". The formatter will allow the user to enter a few different formats and then format it in the way the validator will expect.

We can tweak later if we need to.